package.json

```diff
@@ -21,29 +21,32 @@
     "build-docs-preview-site": "lerna run build-docs-preview-site; rm -r dist/api-docs || true; lerna exec -- mkdir -p ../../dist/api-docs/\\${PWD##*/}; lerna exec -- \"cp -r docs/api/build/html/. ../../dist/api-docs/\\${PWD##*/}/ || true\"; echo 'API documentation: <a href=\"./browser/\">solid-client-authn-browser</a>, <a href=\"./core/\">solid-client-authn-core</a>.' >> dist/api-docs/index.html"
   },
   "dependencies": {
+    "@inrupt/oidc-dpop-client-browser": "file:packages/oidc-dpop-client-browser",
+    "@inrupt/solid-client-authn-browser": "file:packages/browser",
+    "@inrupt/solid-client-authn-core": "file:packages/core",
     "lerna": "^3.22.1"
   },
   "devDependencies": {
-    "@types/jest": "^26.0.13",
+    "@types/jest": "^26.0.12",
```

Comment:
Note that some of these versions are lower, presumably because some of the child packages used to have a lower version. Dependabot should take care of that for future upgrades (also see #318).